doom-modeline-segments.el

```diff
@@ -1353,30 +1353,34 @@ one. The ignored buffers are excluded unless `aw-ignore-on' is nil."
 (advice-add #'winum--install-mode-line :override #'ignore)
 (advice-add #'winum--clear-mode-line :override #'ignore)
 
-(doom-modeline-def-segment window-number
-  (let ((num (cond
-              ((bound-and-true-p ace-window-display-mode)
-               (aw-update)
-               (window-parameter (selected-window) 'ace-window-path))
-              ((bound-and-true-p winum-mode)
-               (setq winum-auto-setup-mode-line nil)
-               (winum-get-number-string))
-              ((bound-and-true-p window-numbering-mode)
-               (window-numbering-get-number-string))
-              (t ""))))
-    (if (and (< 0 (length num))
-             (< (if (active-minibuffer-window) 2 1) ; exclude minibuffer
-                (length (cl-mapcan
-                         (lambda (frame)
-                           ;; Exclude child frames
-                           (unless (and (fboundp 'frame-parent)
-                                        (frame-parent frame))
-                             (window-list)))
-                         (visible-frame-list)))))
-        (propertize (format " %s " num)
-                    'face (if (doom-modeline--active)
-                              'doom-modeline-buffer-major-mode
-                            'mode-line-inactive)))))
+(doom-modeline-def-segment workspace-name
+  "The current workspace name or number.
+Requires `winds.el' to be loaded, or `eyebrowse-mode' or
+`Requires tab-bar-mode' to be enabled."
+  (when-let
+      ((name (cond
+              ((featurep 'winds)
```

Comment:
This doesn't mean winds is enabled.

Reply:
Sorry , Do I need to use `bound-and-true-p`. I am not sure.

Reply:
I don't know how to check if it's enabled. What I concern is this case: if installed both `winds` and `eyebrowse`, but only enabled `eyebrowse`. These codes are incorrect to check. 

BTW, it's not released on melpa, right? What's the advantage of `winds` comparing to `eyebrowse`?